### ISSUE TITLE
Driver won't initialize when run as part of an http trigger function (Azure function)

### ISSUE THREAD

**Issue by stevedep** [seleniumbase/SeleniumBase#2476]
I have a custom docker that I use for selenium testing. Meaning that I install chromedriver as part of the docker file. I then trigger the scraping activity via an Azure Function HTTP trigger call. This works well in nearly all scenarios. 

Today, I had to use the seleniumbase package with uc=True to bypass Cloudflare. It works well when I run it from the exec panel in the docker desktop. But my Python code execution is stuck when it is executed via the Azure Function HTTP trigger.  

I does work without the uc=True parameter. I suspect that the package wants to install the chromedriver, even though it was already installed when I run the code via the exec interface. Perhaps it tries to install it to a different path?

Can I specify the paths? Would this help?

I suspect that my Azure function environment cannot access the paths of the Chromedriver or the Chrome executable. But this is really a guess. I would greatly appreciate any help!

`from seleniumbase import Driver
import json
driver = Driver(log_cdp=True, uc=True)`

**Comment by stevedep** [comment]
Thanks for your quick response! I tried with a trace log, but there is no additional output compared to when it runs successfully. Is there an option for SB or Driver to output verbose logging? 
So the step just keeps running endlessly without an error message or additional logging of any kind. It makes me think that something is stuck deeper in the selbase code. 
Btw, it works great without using the Azure Function python lib. So when I run a Python script using only selbase with uc=True, it works. Therefore, I decided to just schedule the docker instance. I did not really need the http trigger functionality. 
Anyways, it seems the Azure Function limits certain operations or something. I did verify that the user executing the Python code is a root user. Parking this investigation for now, but it would be nice to get this working some day, because it would be great to use selbcase in undetected mode when an http trigger is fired.
